daemon/loadbalancer.go

```diff
@@ -425,17 +425,21 @@ func (d *Daemon) RevNATDump() ([]loadbalancer.L3n4AddrID, error) {
 }
 
 func restoreServiceIDs() {
-	svcMap, _, errors := lbmap.DumpServiceMapsToUserspace(false, false)
+	svcMap, _, errors := lbmap.DumpServiceMapsToUserspace(true, false)
 	for _, err := range errors {
 		log.WithError(err).Warning("Error occured while dumping service table from datapath")
 	}
 
 	for _, svc := range svcMap {
+		if uint32(svc.FE.ID) == uint32(0) {
```

Comment:
Similar to prior comment, indicate why we are skipping ID zero. It took me a long time to realize what ID zero represents. Even better would be to add a wrapper function around this condition to be used throughout this package (which can be done at a later time) to encode the meaning of ID zero.

Reply:
Feel free to do this. As written in the PR, this is minimal code change fix to allow for backporting.